webcompat/views.py

```diff
@@ -201,14 +204,17 @@ def create_issue():
 
 
 @app.route('/issues/<int:number>')
+@cache_policy(private=True, uri_max_age=86400)
 def show_issue(number):
     '''Route to display a single issue.'''
     if g.user:
         get_user_info()
     if session.get('show_thanks'):
         flash(number, 'thanks')
         session.pop('show_thanks')
-    return render_template('issue.html', number=number)
+    content = render_template('issue.html', number=number)
+    response = make_response(content)
+    return response
```

Comment:
nit: Personally I would write `return make_response(content)` to avoid the extra variable here, but that's up to you.